### PR TITLE
Add kube-version and distro on detail panel

### LINKED ISSUE #3332: Display kube version somewhere
**What would you like to be added**:
Some view that shows the detected kubelet version of a cluter.

**Why is this needed**:
Sometimes it is nice to know what version is being run

### LINKED ISSUE #3375: lens/extensions 5+ doesn't provide access to k8s cluster version
Hi lens team, 
My [extension](https://github.com/pashevskii/debug-pods-lens-extension) used clusterStore to get k8s version. It allowed building kubectl command considering a specific version of k8s instance (or block the ability to use some features if k8s doesn't support these ones). 
But, after extensions 5+ release, access to clusterStore was suppressed, moreover, the package doesn't provide alternative ways to get information about clusters. That fully blocks implementing lens5 compatible version of the extension

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

resolves #3332 

Screen Shot 2021-07-12 at 4 36 08 PM

### CODE REVIEW

**Comment by jakolehm** [comment]
IMHO we should add this information to details drawer, not labels.

**Comment by Nokel81** [comment]
Sounds good. What should we do about the distro? Also have it only in the drawer?

File: src/renderer/initializers/catalog-entity-detail-registry.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+export function initCatalogEntityDetailRegistry() {
+  CatalogEntityDetailRegistry.getInstance()
+    .add([
+      {
+        apiVersions: [KubernetesCluster.apiVersion],
+        kind: KubernetesCluster.kind,
+        components: {
+          Details: ({ entity }: CatalogEntityDetailsProps<KubernetesCluster>) => (
+            <>
+              <div className="box grow EntityMetadata">
```
Comment (jakolehm) [inline src/renderer/initializers/catalog-entity-detail-registry.tsx:37]: This should have `<DrawerTitle />` component to match look&feel of other detail panels.

**Comment by Nokel81** [comment]
@jakolehm PTALA?

**Review by jakolehm — APPROVED** [review]

**Pull request merged**